benchmarks/static_map/insert_bench.cu

```diff
@@ -94,13 +85,27 @@ NVBENCH_BENCH_TYPES(static_map_insert,
   .set_name("static_map_insert_unique_occupancy")
   .set_type_axes_names({"Key", "Value", "Distribution"})
   .set_max_noise(defaults::MAX_NOISE)
+  .add_int64_axis("NumInputs", {defaults::N})
```

Comment:
shall we add the `Multiplicity` axis to all test cases?

Reply:
I thought about this too. Multiplicity/Skew are parameters of the input distribution. If we select a unique key distribution, then the Multiplicity/Skew values in those extra columns are completely unrelated. I'd say that is confusing.

Reply:
Good point.

This test is dedicated to `unique` input and we can vary multiplicity in `static_map_insert_uniform_multiplicity`.